FreeRTOS-Plus/Source/Utilities/logging/logging_stack.h

```diff
@@ -37,25 +37,49 @@
 #include "logging_levels.h"
```

Comment:
nit: I think the header include guards conventionally don't have the trailing underscore in FreeRTOS

Reply:
Changed to `LOGGING_STACK`